### PR TITLE
fix WAPI not defined added headless and devtools as options and sendMessage to not existing chats

### PR DESCRIPTION
**Pull request by robertsLando**

### CODE REVIEW

**Comment by robertsLando** [comment]
Fix #212

**Comment by yoavmmn** [comment]
Thanks! looks very good

**Comment by robertsLando** [comment]
@danielcardeenas This make this lib working again

File: src/controllers/browser.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export async function initWhatsapp(
   const browser = await initBrowser(session, headless, devtools);
   const waPage = await getWhatsappPage(browser);
   await waPage.setUserAgent(
-    'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3312.0 Safari/537.36'
+    'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.0 Safari/537.36'
```
Comment (yoavmmn) [inline src/controllers/browser.ts]: Don't you think it's better to use the UA of a more popular "real user" OS?

Comment (robertsLando) [reply src/controllers/browser.ts]: I have use my pc user agent :) If you want I can revert that part to the previous one

Comment (robertsLando) [reply src/controllers/browser.ts]: There seems to be another problem, I have to remove the session folder after each restart or the userAgent is not correctly working and the qr code is not shown, instead I get a page that tells me to update chrome. To fix this I'm tring to use some other flags

**Comment by robertsLando** [comment]
Fixes #202 
Fixes #212

**Comment by robertsLando** [comment]
We should keep wapi updated by looking to PR/Changes on https://github.com/mukulhase/WebWhatsapp-Wrapper/blob/master/webwhatsapi/js/wapi.js

**Comment by robertsLando** [comment]
@danielcardeenas Please check out also new feature to get qrcode image with event :)

**Comment by danielcardeenas** [comment]
I'll keep that QR thing for tomorrow. Got to make the new README

**Pull request closed without merging**